<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QGMDockView</class>
 <widget class="QDockWidget" name="QGMDockView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>155</width>
    <height>343</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>155</width>
    <height>343</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>&amp;View Port</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <property name="verticalSpacing">
     <number>4</number>
    </property>
    <item row="0" column="0">
     <widget class="QLabel" name="labelFPS">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>FPS:</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelFPSValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelDPI">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>DPI:</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="labelDPIValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelLightCam">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Li.Cam:</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelLightCamValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelLightWorld">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Li.Obj:</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="labelLightWorldValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="labelSelPrim">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SelPrim:</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="labelSelPrimValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="labelFuncVal">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>FuncVal:</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLabel" name="labelFuncValValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="labelLabel">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Label ID:</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="labelLabelValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="labelSelMVerts">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SelMVerts:</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="labelSelMVertsValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QLabel" name="labelSelMFaces">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SelMFaces:</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QLabel" name="labelSelMFacesValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="labelSelMPolys">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>SelMPolys:</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="labelSelMPolysValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QLabel" name="labelPositions">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Positions</string>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QLabel" name="labelPositionsValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
      <property name="margin">
       <number>2</number>
      </property>
      <property name="indent">
       <number>-1</number>
      </property>
     </widget>
    </item>
    <item row="15" column="0">
     <widget class="QLabel" name="labelUser">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>User:</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QLabel" name="labelUserValue">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="text">
       <string>-</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
